tests/pytorch/test_nn.py

```diff
@@ -1529,7 +1529,7 @@ def test_typed_linear(feat_size, regularizer, num_bases):
 
     assert th.allclose(y, y_sorted[rev_idx], atol=1e-4, rtol=1e-4)
 
-@parametrize_idtype
+@pytest.mark.parametrize('idtype', [F.int64])
```

Comment:
@mufeili I changed the parameter type because of the last build we ran. I didn't write this test, so I did not want to change it at the function level. 

![image](https://user-images.githubusercontent.com/25559310/202879461-dd36c235-3492-4034-807e-98d8b7a3918b.png)

Reply:
Could you revert this change? The test failed probably due to the randomly generated graph and let's trigger CI again.

Reply:
done. Can you trigger the CI?